src/components/molecules/PollBox/PollBox.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { useCallback, useMemo } from "react";
+import { useForm, useFieldArray } from "react-hook-form";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faPaperPlane, faPlus } from "@fortawesome/free-solid-svg-icons";
+
+import { MAX_POLL_CHOICES } from "settings";
+
+import { getVenueOwners } from "api/admin";
+
+import { PollValues, PollQuestion } from "types/chat";
+
+import { InputField } from "components/atoms/InputField";
+
+import { useVenueId } from "hooks/useVenueId";
+import { useRoles } from "hooks/useRoles";
+import { useUser } from "hooks/useUser";
+
+import "./PollBox.scss";
+
+export interface PollBoxProps {
+  createPoll: (props: PollValues) => void;
+}
+
+const DEFAULT_QUESTION: Partial<PollQuestion> = { name: "" };
+const DEFAULT_VALUES = {
+  topic: "",
+  questions: [DEFAULT_QUESTION, DEFAULT_QUESTION],
+};
+
+export const PollBox: React.FC<PollBoxProps> = ({ createPoll }) => {
+  const venueId = useVenueId();
+  const { userRoles } = useRoles();
+  const { user } = useUser();
+
+  const userId = user?.uid;
+  const isAdmin = Boolean(userRoles?.includes("admin"));
+
+  const { register, control, handleSubmit, reset, watch } = useForm<PollValues>(
+    {
+      defaultValues: DEFAULT_VALUES,
+    }
+  );
```

Comment:
Can we just have undefined? Why do we need default values here, since they are empty?

Reply:
`useForm` recommends setting explicit empty values rather than `undefined`

- https://react-hook-form.com/api/useform
  - > It is encouraged that you set `defaultValues` for all inputs to non-`undefined` values such as the empty string or `null`.

That said, i'm not sure that we do this across the codebase at the moment, and not sure if it's a big issue/what the consequence is if we set these to something else.